src/app/pages/parents/parents.component.ts

```diff
@@ -17,17 +17,15 @@ export class ParentsComponent implements OnInit {
     { def: "phoneNumber", label: "Phone" },
     { def: "email", label: "Email" },
     { def: "workInfo", label: "Work Info" },
-    { def: "children", label: "Children"}
+    { def: "children", label: "Children" }
   ];
 
-  constructor(private service:ParentService) {}
-  
-  
-  parentsData : IParent[] ;
+  constructor(public service: ParentsService) { }
+
+
+  parentsData: IParent[];
```

Comment:
It is good that you made it public, but still 'service', empty lines, and extra space